src/cli/main.cpp

```diff
@@ -82,18 +82,19 @@ int main(const int argc, const char** const argv)
             std::cerr << "Failed to create daemon.";
             return EXIT_FAILURE;
         }
-/*➕
-        // Demo of daemon's node command client, sending a command to node 42, 43 & 44.
+
+#if 1  // NOLINT
```

Comment:
?

Reply:
If you asking "why `#if 1` ?" then remember I shared that I'm making series of "smaller" consecutive PRs (for the sake of more manageable code reviewing) from my private branch, where everything is already done (including file server). In my branch I have 4-5 of such sections, each calling different parts of SDK. Also, "cli" executable was/is out of requirements currently, but I need a "vehicle" to do real rides. So its "quality" is not my focus right now.